### ISSUE TITLE
ModuleNotFoundError: No module named 'pkg_resources'

### ISSUE THREAD

**Issue by FlorentinD** [jiffyclub/palettable#59]
First of all, thank you for this extensive collection!

Using version `3.3.0`.
We hit the following issue on trying to import a palette from the package 

```
>       from palettable.wesanderson import Moonrise1_5

tests/test_colors.py:75: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../.venv311/lib/python3.11/site-packages/palettable/__init__.py:10: in <module>
    from . import colorbrewer
../.venv311/lib/python3.11/site-packages/palettable/colorbrewer/__init__.py:3: in <module>
    from .colorbrewer import *
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from __future__ import absolute_import
    from __future__ import print_function
    
    import json
    import webbrowser
    
>   from pkg_resources import resource_string
E   ModuleNotFoundError: No module named 'pkg_resources'

../.venv311/lib/python3.11/site-packages/palettable/colorbrewer/colorbrewer.py:7: ModuleNotFoundError
```

Our workaround was to install `setuptools`, but reading up on https://setuptools.pypa.io/en/latest/pkg_resources.html, it should be possible to replace the usage of `pkg_resources` as well.
However, I cannot even find the import in the source code?

**Comment by jiffyclub** [comment]
~Hi, thanks for the report! It looks like [importib.resources](https://docs.python.org/3/library/importlib.resources.html#module-importlib.resources) provides this functionality in the standard library now, I'll have to look into updating that.~

**Comment by jiffyclub** [comment]
Actually, looking deeper into this, this was fixed in #46. Those changes were released in version 3.3.1, so if you update to the latest version you shouldn't run into this issue.
